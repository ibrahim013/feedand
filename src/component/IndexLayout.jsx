import React from 'react';

const IndexLayout = () => (
    <div>
        <h1>welcome</h1>
    </div>
)

export default IndexLayout;