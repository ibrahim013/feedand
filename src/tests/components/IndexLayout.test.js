import React from 'react';
import { shallow } from 'enzyme';
import IndexLayout from '../../component/IndexLayout.jsx'

test('should render IndexLayout correctly', () => {
  const wrapper = shallow(<IndexLayout />);
  expect(wrapper).toMatchSnapshot();
});