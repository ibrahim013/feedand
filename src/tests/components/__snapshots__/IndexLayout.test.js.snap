// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render IndexLayout correctly 1`] = `
<div>
  <h1>
    welcome
  </h1>
</div>
`;
